### ISSUE TITLE
[OH3] constantly restarting (generates hs_err logfiles)

### ISSUE THREAD

**Issue by GumbyMan82** [openhab/openhab-core#2083]
Config:
* Raspi 3+
* Clean installation with openhabian 1.6

Today my openhab went into a constant bootloop.
frontail logs showed always that all items get initialized etc.. so the same as if you command
systemctl stop openhab.service
systemctl start openhab.service

I then found out that in /var/lib/openhab a lot of logfiles have been generated.
It seems they are created by the JAVA and all start like this:

```
A fatal error has been detected by the Java Runtime Environment:
SIGBUS (0x7) at pc=0x76aabd84, pid=13087, tid=13355

 JRE version: OpenJDK Runtime Environment Zulu11.43+88-CA (11.0.9+11) (build 11.0.9+11-LTS)
 Java VM: OpenJDK Client VM Zulu11.43+88-CA (11.0.9+11-LTS, mixed mode, serial gc, linux-arm)
 Problematic frame:
 V  [libjvm.so+0x57cd84]  _Copy_conjoint_jints_atomic+0x24

 No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again

 If you would like to submit a bug report, please visit:
   http://www.azulsystems.com/support/


---------------  S U M M A R Y ------------

Command Line: -XX:-UsePerfData -Dopenhab.home=/usr/share/openhab -Dopenhab.conf=/etc/openhab -Dopenhab.runtime=/usr/share/openhab/runtime -Dopenhab.userdata=/var/lib/openhab -Dopenhab.logdir=/var/log/openhab -Dfelix.cm.dir=/var/lib/openhab/config -Djava.library.path=/var/lib/openhab/tmp/lib -Djetty.host=0.0.0.0 -Djetty.http.compliance=RFC2616 -Dnashorn.args=--no-deprecation-warning -Dorg.ops4j.pax.web.listening.addresses=0.0.0.0 -Dorg.osgi.service.http.port=8080 -Dorg.osgi.service.http.port.secure=8443 -Djava.awt.headless=true -Xms192m -Xmx320m --add-reads=java.xml=java.logging --add-exports=java.base/org.apache.karaf.specs.locator=java.xml,ALL-UNNAMED --patch-module=java.base=/usr/share/openhab/runtime/lib/endorsed/org.apache.karaf.specs.locator-4.2.7.jar --patch-module=java.xml=/usr/share/openhab/runtime/lib/endorsed/org.apache.karaf.specs.java.xml-4.2.7.jar --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.naming/javax.naming.spi=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED --add-exports=java.base/sun.net.www.protocol.http=ALL-UNNAMED --add-exports=java.base/sun.net.www.protocol.https=ALL-UNNAMED --add-exports=java.base/sun.net.www.protocol.jar=ALL-UNNAMED --add-exports=jdk.xml.dom/org.w3c.dom.html=ALL-UNNAMED --add-exports=jdk.naming.rmi/com.sun.jndi.url.rmi=ALL-UNNAMED -Dkaraf.instances=/var/lib/openhab/tmp/instances -Dkaraf.home=/usr/share/openhab/runtime -Dkaraf.base=/var/lib/openhab -Dkaraf.data=/var/lib/openhab -Dkaraf.etc=/var/lib/openhab/etc -Dkaraf.log=/var/log/openhab -Dkaraf.restart.jvm.supported=true -Djava.io.tmpdir=/var/lib/openhab/tmp -Djava.util.logging.config.file=/var/lib/openhab/etc/java.util.logging.properties -Dkaraf.startLocalConsole=false -Dkaraf.startRemoteShell=true org.apache.karaf.main.Main

Host: rev 4 (v7l), 4 cores, 974M, Raspbian GNU/Linux 10 (buster)
Time: Thu Jan  7 21:21:47 2021 CET elapsed time: 98.111351 seconds (0d 0h 1m 38s)

---------------  T H R E A D  ---------------

Current thread (0x5bd3d000):  JavaThread "items-4" [_thread_in_vm, id=13355, stack(0x57d79000,0x57dc9000)]
```

all logfiles can be downloaded here:
https://drive.google.com/file/d/1DaRljjQ73n9aQazkfus05G1hUgTtytBW/view?usp=sharing

Restarting openhab service did not help - even cleaning the cache first did not help.

Edit:
Rebooting the raspi did help

**Comment by GumbyMan82** [comment]
Another (useful?) hint:
This is the log from the faulty state:
![image](https://user-images.githubusercontent.com/40233411/103946940-a1879300-5137-11eb-9fd4-07596762d204.png)

And this is from the working state:
![image](https://user-images.githubusercontent.com/40233411/103946997-b2380900-5137-11eb-88cc-763de73c9c01.png)

**Comment by MHerbst** [comment]
This is a duplicate to #2032 

I don't think that it is caused by the multiple interfaces, because the crash happens while writing information to rrd4j that has nothing to do with the interfaces. 

If you have ZRAM configured, you could try to disable it and test what happens.

**Comment by GumbyMan82** [comment]
I will try my best. 
Since it happened once within two weeks, it's a bit hard to reproduce.

**Comment by MHerbst** [comment]
Yes, that's hard :-(. In my opinion, this is probably a bug in the Java interpreter itself. I think it would be useful to report the error also to Azul.

**Comment by kaikreuzer** [comment]
Closing as duplicate of https://github.com/openhab/openhab-core/issues/2032

**Comment by phixenl** [comment]
Hi,

I can confirm having the same issues, happens once a week sofar (clean openhabian install).
```
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [libjvm.so+0x57cd84]  _Copy_conjoint_jints_atomic+0x24
C  0x5842e13c
J 1688  jdk.internal.misc.Unsafe.copyMemory0(Ljava/lang/Object;JLjava/lang/Object;JJ)V java.base@11.0.9 (0 bytes) @ 0x744c30f0 [0x744c3040+0x000000b0]
J 7389 c1 java.nio.DirectByteBuffer.put([BII)Ljava/nio/ByteBuffer; java.base@11.0.9 (151 bytes) @ 0x746f7858 [0x746f7680+0x000001d8]
J 7390 c1 org.rrd4j.core.RrdNioBackend.write(J[B)V (65 bytes) @ 0x746e8360 [0x746e80a0+0x000002c0]
J 7464 c1 org.rrd4j.core.RrdPrimitive.writeDouble(ID)V (26 bytes) @ 0x74714968 [0x747148a0+0x000000c8]
```

Recently I noticed issues with creating the items.rrd file, permission dendied errors, like these:

`2021-02-02 07:18:31.850 [ERROR] [d4j.internal.RRD4jPersistenceService] - Could not create rrd4j database file '/var/lib/openhab/persistence/rrd4j/item1.rrd': /var/lib/openhab/persistence/rrd4j/item1.rrd (Permission denied)`

I repaired it with changing owner:group and permission. but then I got warning message and i decided to delete the affected items.rrd files. For a while everything went smoothly untill last night.

I don't know what happens first as the openhab log files are overwritten before i noticed the crash. Could be i get permissions errors due to the crash...

Perhaps this information does ring a bel.
